_components/accordions.md

```diff
@@ -139,29 +139,21 @@ lead: Intro text on what is included in this section and how to use it. No more
     Documentation
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
-    <h4 class="usa-heading">Use</h4>
+    <h4 class="usa-heading">Usability</h3>
+    <h5>When to use</h4>
```

Comment:
Same as above but this time h5 and h4.

So this should either be `<h5>When to use</h5>` or `<h4>When to use</h4>` depending on what's needed for the design.